docs/getting-started/linux-installer.md

```diff
@@ -230,11 +246,81 @@ applicable for `td-agent` versions 4.1 or newer):
   sudo systemctl restart td-agent
   ```
 
+### Auto Instrumentation
+
+#### Installation
+
+To install the Collector and the Splunk OpenTelemetry Auto Instrumentation for
+Java packages, run the installer script with the `--with-instrumentation`
+option:
+```sh
+curl -sSL https://dl.signalfx.com/splunk-otel-collector.sh > /tmp/splunk-otel-collector.sh && \
+sudo sh /tmp/splunk-otel-collector.sh --with-instrumentation --realm SPLUNK_REALM -- SPLUNK_ACCESS_TOKEN
+```
+
+To automatically define the optional `deployment.environment` resource
+attribute at installation time, run the installer script with the
+`--deployment-environment VALUE` option (replace `VALUE` with the desired
+attribute value, e.g. `prod`):
+```sh
+curl -sSL https://dl.signalfx.com/splunk-otel-collector.sh > /tmp/splunk-otel-collector.sh && \
+sudo sh /tmp/splunk-otel-collector.sh --with-instrumentation --deployment-environment VALUE --realm SPLUNK_REALM -- SPLUNK_ACCESS_TOKEN
+```
+
+**Note:** After successful installation, the Java application(s) on the host need to be
+manually started/restarted for auto instrumentation to take effect.
+
+#### Post-Install Configuration
+
+- The `/etc/ld.so.preload` file will be automatically updated with the default
+  path to the installed instrumentation library
+  (`/usr/lib/splunk-instrumentation/libsplunk.so`).  If necessary, custom
+  library paths can be manually added to this file.
+- The `/usr/lib/splunk-instrumentation/instrumentation.conf` file can be
+  manually configured for resource attributes and other options.  If the
+  `--deployment-environment VALUE` installer script option was specified,
+  the `deployment.environment=VALUE` resource attribute will be automatically
+  added to this file.
+
+See [Linux Java Auto Instrumentation](https://github.com/signalfx/splunk-otel-collector/tree/main/instrumentation#linux-java-auto-instrumentation)
+for more details.
+
+**Note:** After any configuration changes, the Java application(s) on the host
+need to be manually started/restarted to source the updated values.
+
+#### Upgrade
+
+To upgrade the Instrumentation package, run the following commands on your
+system (requires `root` privileges):
+- Debian:
+  ```sh
+  sudo apt-get update
+  sudo apt-get --only-upgrade splunk-otel-auto-instrumentation
+  ```
+  **Note:** You may be prompted to keep or overwrite the configuration file at
+  `/usr/lib/splunk-instrumentation/instrumentation.conf`.
```

Comment:
Should the user keep or overwrite the file if prompted?

Reply:
The decision is up to the user or determined by the how their system is configured.